Resources/Prototypes/EntityLists/Tools/cow.yml

```diff
@@ -0,0 +1,10 @@
+﻿- type: entityList
+  id: Cowtools
+  entities:
+  - Haycutters
+  - Moodriver
+  - Wronch
+  - Cowbar
+  - Mooltitool
+  - Cowelder
+  - Milkalyzer
```

Comment:
Sorry bud you are late to the party https://github.com/space-wizards/space-station-14/blob/677706b117959823c95986f39f8faf6627458211/Resources/Prototypes/EntityLists/Cow/cow_tools.yml